### PR TITLE
Adding dennie's bio info

### PR DESCRIPTION
**Pull request by dennie**
Adding in my bio info, please review!

### CODE REVIEW

File: bio/dennie_bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Bio for dennie
+- lives in SF
+- hometown: SD
+- coffee lover
```
Comment (michaelkemp2000) [inline bio/dennie_bio.md:4]: I like coffee too.  Do you take cream?

**Comment by treinartz** [comment]
adding comment

**Comment by treinartz** [comment]
testing plus sign

**Comment by eabedrapo** [comment]
well done!

**Comment by shoyambhu** [comment]
test

**Pull request merged**